pkg/authorization/api/v1/types.go

```diff
@@ -70,6 +76,8 @@ type RoleBinding struct {
 	RoleRef kapi.ObjectReference `json:"roleRef"`
 }
 
+type NamedRoles []NamedRole
```

Comment:
Because this makes conversions clear.  A conversion is between two types,
and the name for an alias is unambiguous (Convert_foo_to_bar).

On Thu, Jul 14, 2016 at 10:19 AM, David Eads <notifications@github.com>
wrote:

> In pkg/authorization/api/v1/types.go
> <https://github.com/openshift/origin/pull/9838#discussion_r70813159>:
>
> > @@ -70,6 +76,8 @@ type RoleBinding struct {
> >  	RoleRef kapi.ObjectReference `json:"roleRef"`
> >  }
> >
> > +type NamedRoles []NamedRole
>
> You're doing this a lot. Why?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9838/files/ffdbff08f0e5a383e87d21f1f765210dc9139a97#r70813159>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p4eKxz1V7iPULrjcxuCYjneEKmj7ks5qVkVjgaJpZM4JMCT8>
> .
>